tools/ci_build/github/linux/java_linux_final_test.sh

```diff
@@ -20,23 +20,22 @@ EXIT_CODE=1
 
 uname -a
 
-cd "$BINARY_DIR/final-jar"
-
-mkdir test
+cd "$BINARY_DIR/onnxruntime-java"
+rm -f *.asc
+rm -f *.sha256
+rm -f *.sha512
+rm -f *.pom
+ls
+cd ..
+mkdir tests
+cd tests
+jar xf ../onnxruntime-java/testing.jar
+rm -f ../onnxruntime-java/testing.jar
+echo "Java Version"
+java -version
 
 echo "Directories created"
 echo  "Library path:" "$LD_LIBRARY_PATH"
 
-pushd test
-jar xf "$BINARY_DIR/final-jar/testing.jar"
-popd
-
-curl -O -sSL https://oss.sonatype.org/service/local/repositories/releases/content/org/junit/platform/junit-platform-console-standalone/1.6.2/junit-platform-console-standalone-1.6.2.jar
```

Comment:
Mainly, this change is to avoid directly downloading such files.  Use maven package manager instead.